### ISSUE TITLE
`nfd-worker` errors reading `speed` parameter for paravirtualized devices

### ISSUE THREAD

**Issue by mantramantra12** [kubernetes-sigs/node-feature-discovery#1556]
**What happened**:
The `nfd-worker` pods on my cluster get this error every time they perform feature discovery:

```
"failed to read net iface attribute" err="read /host-sys/class/net/ifb0/speed: invalid argument" attributeName="speed"
```

I believe this is because the devices are paravirtual and so the `speed` parameter is nonsensical in this context. The file still exists however I think because the kernel puts it there.

**What you expected to happen**:
Feature discovery completes without error

**How to reproduce it (as minimally and precisely as possible)**:
Run feature discovery on a node with paravirtual devices connected

**Anything else we need to know?**:
N/A

**Environment**:
- Kubernetes version (use `kubectl version`): 1.27.7
- Cloud provider or hardware configuration: Proxmox ubuntu VM
- OS (e.g: `cat /etc/os-release`): Ubuntu 22.04.3 LTS
- Kernel (e.g. `uname -a`): Linux <hostname> 5.15.0-91-generic 101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by marquiz** [comment]
Fixed by #1557 
/close

**Comment by weikinhuang** [comment]
I'm still seeing this for the localhost interface

```
E0321 23:00:33.543500       1 network.go:152] "failed to read net iface attribute" err="read /host-sys/class/net/lo/speed: invalid argument" attributeName="speed"
```

**Comment by marquiz** [comment]
@weikinhuang which version of nfd are you running? The fix hasn't been released, yet. It'll be in the upcoming v0.16 release

**Comment by weikinhuang** [comment]
@marquiz oh, I thought the fix was in 0.15.2. I am on 0.15.3. If that's the case thanks for letting me know!
